Makefile

```diff
@@ -45,7 +45,7 @@ vet:
 
 test:
 	# Runs only unit tests
-	go test -race -tags=\!integration $$(go list ./... | grep -v /vendor/ | grep -v /cmd/)
+	go test -race -tags=\!integration\!property $$(go list ./... | grep -v /vendor/ | grep -v /cmd/)
```

Comment:
I would leave property based tests in here.  We want them to run as part of Travis CI.  This will make them no longer run